cmd/rekor-cli/app/log_info.go

```diff
@@ -43,7 +43,7 @@ type logInfoCmdOutput struct {
 	TreeSize       int64
 	RootHash       string
 	TimestampNanos uint64
-	TreeID         int64
+	TreeID         string
```

Comment:
Maybe use `json.Number`? https://pkg.go.dev/encoding/json#Number

Reply:
Weirdly I'm still seeing the same issue with this even though it should be a string...